src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
+
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private static final String SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            String fileContent = readFile(fromFileName);
+            String[] results = processFileContent(fileContent);
+            writeToFile(toFileName, results);
+        } catch (IOException e) {
+            throw new RuntimeException("Error processing files", e);
+        }
```

Comment:
It is better to check the exception in `readFile` method
```suggestion
            String fileContent = readFile(fromFileName);
            String[] results = processFileContent(fileContent);
            writeToFile(toFileName, results);
```